assets/src/pages/types.ts

```diff
@@ -2,7 +2,7 @@ import { TrackEncoding } from "@membraneframework/membrane-webrtc-js";
 
 // todo Change TrackEncoding to something like EncodingType in "@membraneframework/membrane-webrtc-js"
 const EncodingValues = ["l", "m", "h"] as const;
-type EncodingType = typeof EncodingValues[number];
+type EncodingType = typeof EncodingValues[number]; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
What is the purpose of this type variables? Instead of disabling linter, either remove them or export them